packages/cozy-pouch-link/src/CozyPouchLink.spec.js

```diff
@@ -323,4 +363,32 @@ describe('CozyPouchLink', () => {
       expect(link.onLogin()).rejects.toThrow()
     })
   })
+
+  describe('index creation', () => {
+    let spy
+
+    afterEach(() => {
+      spy.mockRestore()
+    })
+
+    it('uses the default index aka the from the sort', async () => {
```

Comment:
```suggestion
    it('uses the default index, the one from the sort', async () => {
```

I do not think that aka was OK to use here.